src/config.rs

```diff
@@ -373,6 +375,36 @@ fn merge_configuration_files(
     output.files = first_package.files;
     output.variables = first_package.variables;
 
+    // Defaults package target type to symlink if
+    // enable_symlink_by_default = true and
+    // package is not explicitly set as template
+    if let Some(variables) = global.variables {
+        if let Some(enable_symlink_by_default) =
+            variables
+                .get("enable_symlink_by_default")
+                .and_then(toml::Value::as_bool) {
+                    if enable_symlink_by_default {
```

Comment:
maybe it will be cleaner to do something along the lines of `global.settings.get("enable_symlink_by_default") == Some(toml::Value::Bool(true))`